client/src/components/dashboard/DashboardEditor.jsx

```diff
@@ -181,6 +183,22 @@ export default class DashboardEditor extends Component {
     this.props.onUpdateEntities(newEntities);
   }
 
+  handleNameFocus() {
```

Comment:
Would it be possible to use the placeholder `input` attribute (as described in https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input) instead of hooking into `onFocus`?